smart_card_connector_app/src/window-apps-displaying.js

```diff
@@ -112,14 +144,17 @@ function onUpdateListener(appListArg) {
 }
 
 GSC.ConnectorApp.Window.AppsDisplaying.initialize = function() {
+  var passedData = GSC.PopupWindow.Client.getData();
+
   logger.fine('Registering listener on connected apps update');
   // FIXME(emaxx): Do unsubscription too.
-  // FIXME(emaxx): Use GSC.ObjectHelpers.extractKey to ensure that the expected
-  // object is passed to the window.
-  var data = GSC.PopupWindow.Client.getData()['clientAppListUpdateSubscriber'];
+  var data = extractKey(passedData, 'clientAppListUpdateSubscriber');
```

Comment:
nit: Collapse this with the following line (as the "data" temporary variable doesn't bring much information).